src/styles.scss

```
@@ -23,6 +23,11 @@ $content-max-width: 1400px;
     max-width: min(72rem, calc(100vw - 2rem));
   }
 
+  .tooltip-panel--warning-wide {
+    @apply border-yellow-500 whitespace-pre-line;
+    max-width: min(54rem, calc(100vw - 2rem));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)